### PR TITLE
Add category for third-party middleware

### PR DESCRIPTION
**Pull request by titouv**
This PR includes the following categories for the page listing third-party middleware:  
-  Authentication  
-  Validators  
-  OpenAPI  

What can be discussed is the order of these categories; for now, it is organized by the number of entries in each category.
 We can also add more categories, but I have not found any meaningful groups for the rest of the list.

### CODE REVIEW

File: docs/middleware/third-party.md [hunk 1]
```diff
@@ -3,32 +3,43 @@
...
+
+- [Zod OpenAPI](https://github.com/honojs/middleware/tree/main/packages/zod-openapi)
+- [Scalar API Reference](https://github.com/scalar/scalar/tree/main/packages/hono-api-reference)
+- [Swagger UI](https://github.com/honojs/middleware/tree/main/packages/swagger-ui)
+
+### Others
+
+- [Bun Transpiler](https://github.com/honojs/middleware/tree/main/packages/bun-transpiler)
 - [esbuild Transpiler](https://github.com/honojs/middleware/tree/main/packages/esbuild-transpiler)
 - [ESLint Config](https://github.com/honojs/middleware/tree/main/packages/eslint-config)
```
Comment (yusukebe) [inline docs/middleware/third-party.md:35]: This `ESLint Config` is not middleware for developers. We are mainly using it internally. Please remove this line.

**Comment by yusukebe** [comment]
@titouv 

This is awesome! For now, it's ok to go with this order. I've left a comment. Please check it.

File: docs/middleware/third-party.md (hunk 1 above)
Comment (titouv) [reply docs/middleware/third-party.md]: It was there before my categorization commit, so I didn't want to remove anything.
This is now done in  https://github.com/honojs/website/pull/468/commits/47c05fb955d1acd2e480dbe27664a8243a2d4786

Comment (yusukebe) [reply docs/middleware/third-party.md]: @titouv 

Ah, thank you!

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@titouv 

Thank you very much! I'm merging.

**Pull request merged**